Source/IntegrationTests/IntegrationTests.csproj

```diff
@@ -47,6 +47,16 @@
         <Content Include="..\..\node_modules\**\*.*" LinkBase="TestFiles\LitTests\LitTest\node_modules">
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
         </Content>
+        <Content Update="..\..\Test\git-issues\github-issue-4176.dfy">
```

Comment:
Please remove this change. We don't need this here, tests are discovered automatically. It was probably added by Rider.